app/src/main/kotlin/com/saltedge/authenticator/features/authorizations/list/AuthorizationsListInteractorV2.kt

```diff
@@ -168,10 +169,11 @@ class AuthorizationsListInteractorV2(
 
     private fun processEncryptedAuthorizationsResult(encryptedList: List<AuthorizationResponseData>) {
         contract?.coroutineScope?.launch(defaultDispatcher) {
-            val splitList = encryptedList.partition { it.status.isFinalStatus }
-            val finishedData = prepareFinishedAuthorizationData(splitList.first)
+            val notClosedItems = encryptedList.filterNot { KEY_STATUS_CLOSED == it.status  }
+            val splitList: Pair<List<AuthorizationResponseData>, List<AuthorizationResponseData>> = notClosedItems.partition { it.status.isFinalStatus }
```

Comment:
`= encryptedList.partition { it.status.isFinalStatus && !it.status.isClosed }`?

Reply:
In this way:
`encryptedList.filterNot { it.status.isClosed }.partition { it.status.isFinalStatus }`